legacy/signals/signals.go

```diff
@@ -0,0 +1,61 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package signals
+
+import (
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/sirupsen/logrus"
+)
+
+// sigMap holds all signals the Watch() function should terminate
+// when encountering. The key holds the string representation
```

Comment:
This first sentence sounds odd but I don't know the best way to reword it. Can you try to reword it?

Reply:
Looking at the rest of this code, it looks like all of these signals are the ones that should kill (terminate) the auditor (main process). In that case a better name might be `exitSignals`.

Reply:
Ack, re-wording